src/command_validate.cc

```diff
@@ -70,6 +70,55 @@ auto parse_loop(const sourcemeta::core::Options &options) -> std::uint64_t {
   }
 }
 
+// validate instance in a loop to measure avg and stdev
+auto run_loop(sourcemeta::blaze::Evaluator &evaluator,
+              const sourcemeta::blaze::Template &schema_template,
+              const sourcemeta::core::JSON &instance,
+              const std::filesystem::path &instance_path,
+              const int64_t instance_index, const uint64_t loop) -> bool {
+  bool result = true;
+  double sum = 0.0, sum2 = 0.0, empty = 0.0;
+
+  // Overhead evaluation, if not to optimize out!
+  for (auto index = loop; index; index--) {
+    const auto start{std::chrono::high_resolution_clock::now()};
+    const auto end{std::chrono::high_resolution_clock::now()};
+    empty += (double)(std::chrono::duration_cast<std::chrono::nanoseconds>(
```

Comment:
Can you do `static_cast` as a more C++ idiom instead of `(double)` everywhere in the function?

Reply:
Dunno, I do not know about `static_cast`.
My knowledge of C++ is over 20 years old, and I was not planning to learn the language in depth right now. Having a few cast to avoid compiler warnings did not stuck me as very bad.